README.md

```diff
@@ -122,14 +211,15 @@ Both virtual host and virtual path can be specified in environment variable `VIR
 |\*/\*.js|example.com/abc.js, example.org/path/abc.js|example.com/abc.css|
 |\*/\*.do/|example.com/abc.do/, example.org/path/abc.do/|example.com/abc.do|
 |\*/path/\*.php|example.com/path/abc.php|example/abc.php, example.com/root/abc.php|
-|\*.example.com/\*.jpg|www.example.com/abc.jpg, abc.exampe.com/123.jpg|example.com/abc.jpg|
+|\*.example.com/\*.jpg|www.example.com/abc.jpg, abc.example.com/123.jpg|example.com/abc.jpg|
 |\*/path, \*/path/|example.com/path, example.org/path/||
 |example.com:90, https://example.com|example.com:90, https://example.com||
 
-Note: The sequence of the acl rules generated based on VIRTUAL_HOST are randomly. In HAProxy, when an acl rule with a wide scope(e.g. *.example.com) is put before a rule with narrow scope(e.g. web.example.com), the narrow rule will never be reached. As a result, if the virtual hosts you set have overlapping scopes, you need to use `VIRTUAL_HOST_WEIGHT` to manually set the order of acl rules, namely, giving the narrow virtual host a higher weight than the wide one.
+**Note**:
+1. The sequence of the acl rules generated based on VIRTUAL_HOST are randomly. In HAProxy, when an acl rule with a wide scope(e.g. *.example.com) is put before a rule with narrow scope(e.g. web.example.com), the narrow rule will never be reached. As a result, if the virtual hosts you set have overlapping scopes, you need to use `VIRTUAL_HOST_WEIGHT` to manually set the order of acl rules, namely, giving the narrow virtual host a higher weight than the wide one.
+2. Any services have exactly the same VIRTUAL_HOST environment variable settings will be considered and merged in to once service. It may be useful for some testing scenario.
```

Comment:
Every service that has the same VIRTUAL_HOST environment variable setting will be considered and merged into one single service. It may be useful for some testing scenario.